src/mscorlib/Resources/Strings.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
+    Microsoft ResX Schema 
+    
```

Comment:
A bunch of stray whitespace additions in the file.

Reply:
I know, but VS did it automatically when using the designer to add the resource.  I tend to not try to fight VS, because then the next time you just need to fight VS again.  It's a losing battle IMO.